SECURITY.md

```diff
@@ -0,0 +1,106 @@
+
+
+## Proposed Email Content
+
+Provide a descriptive subject line and in the body of the email include the following information:
+
+*   Basic identity information, such as your name and your affiliation or company.
+*   Detailed steps to reproduce the vulnerability  (POC scripts, screenshots, and logs are all helpful to us).
+*   Description of the effects of the vulnerability on this project and the related hardware and software configurations, so that the maintainers can reproduce it.
+*   How the vulnerability affects this project's usage and an estimation of the attack surface, if there is one.
+*   List other projects or dependencies that were used in conjunction with this project to produce the vulnerability.
+
+
+## When to report a vulnerability
+
+*   When you think this project has a potential security vulnerability.
+*   When you suspect a potential vulnerability but you are unsure that it impacts this project.
+*   When you know of or suspect a potential vulnerability on another project that is used by this project.
+
+
+## Patch, Release, and Disclosure
+
+The maintainers will respond to vulnerability reports as follows:
+
+1. The maintainers will investigate the vulnerability and determine its effects and criticality.
+2. If the issue is not deemed to be a vulnerability, the maintainers will follow up with a detailed reason for rejection.
+3. The maintainers will initiate a conversation with the reporter within 3 business days.
+4. If a vulnerability is acknowledged and the timeline for a fix is determined, the maintainers will work on a plan to communicate with the appropriate community, including identifying mitigating steps that affected users can take to protect themselves until the fix is rolled out.
+5. The maintainers will also create a [Security Advisory](https://docs.github.com/en/code-security/repository-security-advisories/publishing-a-repository-security-advisory) using the [CVSS Calculator](https://www.first.org/cvss/calculator/3.0). The maintainers make the final call on the calculated CVSS; it is better to move quickly than making the CVSS perfect. Issues may also be reported to [Mitre](https://cve.mitre.org/) using this [scoring calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator). The draft advisory will initially be set to private.
+6. The maintainers will work on fixing the vulnerability and perform internal testing before preparing to roll out the fix.
+7. The maintainers will provide early disclosure of the vulnerability by emailing a security mailing list if such exists. Distributors can initially plan for the vulnerability patch ahead of the fix, and later can test the fix and provide feedback to the maintainer team. See the section **Early Disclosure to This Project's Distributors List** for details about how to join this mailing list.
```

Comment:
I'm concerned about having this in the .github template for this org. I'm going to guess that out of the ~200 repos in this org, that only a few, if any will have this distributor's list. Maybe we leave it out of this template and have something separate that we use for any of the bigger projects that have this?